website/content/en/preview/AWS/instance-types.md

```diff
@@ -10351,8 +11551,10 @@ below are the resources available with some assumptions and after the instance o
 #### Labels
  | Label | Value |
  |--|--|
+ |karpenter.k8s.aws/instance-category|u|
```

Comment:
Is the family u or u-6?

Reply:
I think the category is still `u` and the family is `u-6tb1`. The `6tb` is more of a description of the amount of memory which you can select already with the instance-memory label.